src/DotVVM.Framework/Binding/BindingProperties.cs

```diff
@@ -137,10 +137,10 @@ public ParsedExpressionBindingProperty(Expression expression)
     /// <summary>
     /// Contains <see cref="System.Linq.Expressions.Expression"/> instance that represents code converted to be evaluated as binding (type conversions applied, ...). 
     /// </summary>
-    public sealed class CastedExpressionBindingProperty
+    public sealed class CastExpressionBindingProperty
     {
         public readonly Expression Expression;
-        public CastedExpressionBindingProperty(Expression expression)
+        public CastExpressionBindingProperty(Expression expression)
```

Comment:
Ah, this is unfortunate... I would like to word Casted to exist, since CastExpression looks like it represents the cast (a type conversion), not that it's an expression that is already converted. If nobody complains, I'd leave it as is. If a grammar nazi will complain, I'd change this to `ConvertedExpressionBindingProperty`.

Reply:
I renamed it back